packages/wonder-blocks-tooltip/util/is-obscured.test.js

```diff
@@ -123,9 +132,80 @@ describe("isObscured", () => {
 
             // Assert
             expect(result).toBeTruthy();
-            // Reset the document method to avoid side-effects.
+            done();
+        };
+
+        arrange(actAndAssert);
+    });
+
+    test("element is not obscured, but elementFromPoint returns parent or child, returns false", (done) => {
```

Comment:
I though this test was part of a different PR.  I hope I'm not losing it.

Reply:
You're not...must've got merged forward. Working on rebasing this onto the already reviewed things. Sorry about that.